example/lib/operations.dart

```diff
@@ -216,6 +216,15 @@ Map<String, dynamic> operations = {
     "fees": 5000,
     "changeAddress": "D9pvhnWknRza2HTXhY5WT29D4kvYzTZQAF",
   },
+
+  'MATIC': {
+    "chainID": "0x01",
```

Comment:
This is ethereum chain id https://chainlist.org/ .
- verify check if the ethereum implementation we already have will work for polygon just by changing the chain id.
- Verify for arbitum and optimism as well.

Reply:
I tried it out with polygon chain Id(base64). I got this error 
`invalid chain id for signer`